### DISCUSSION TITLE
Where is the download link to DMUC-GUI v1.01a?

### DISCUSSION THREAD

**Discussion by tcrnottingham** [CarlGao4/Demucs-Gui#38] [Q&A]
Hi
I can't find as link to v1.01a

**Comment by CarlGao4** [comment]
1.1a1 hasn't been released yet. I just used it to show that it is a later version

**Reply by tcrnottingham** [reply]
Hi
I've downloaded 1.0 which appears to have been updated with more parameters and a progress bar.
Unfortunately after importing a file for splitting the screen status shows 'writing' but doesn't write anything out to the path.

Any thoughts about this?

Kind regards

Clive Morris

**Reply by CarlGao4** [reply]
Please attach the log file (you can open the folder from the menu bar). You can remove some sensitive information (like the file path containing your username, but only remove your username in this case) before uploading it.
